doc/api/packages.md

```diff
@@ -45,29 +44,31 @@ future-proof the package in case the default type of Node.js ever changes, and
 it will also make things easier for build tools and loaders to determine how the
 files in the package should be interpreted.
 
-### Package scope and file extensions
+### Package boundaries and file extensions
 
 A folder containing a `package.json` file, and all subfolders below that folder
 until the next folder containing another [`package.json`][], are a
-_package scope_. Package scopes do not carry through `node_modules` folders.
+_package boundary_. Package boundaries do not carry through `node_modules`
+folders, which are effectively treated as if they contained an empty
+`package.json`.
```

Comment:
We can just delete this whole paragraph. (The suggest tool won’t let me propose this.)